i18n/chs/src/launch.i18n.json

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+// Do not edit this file. It is machine generated.
+{
+	"yes": "Yes",
+	"no": "No",
+   "build.failed.continue.anyway": "Build failed. Do you want to continue anyway?",
+   "cannot.op.no.launch.config.targets": "Cannot '{0}' because there is no launch configuration set and the list of launch targets is empty. Double check the makefile configuration and the build target.",
```

Comment:
Just a suggestion. The loc team is inconsistent with punctuation across languages. I have a PR open on CMake Tools where I move the `'` into the substitution strings to avoid this.  I recommend the same here.

Reply:
Ok, got it.